src/main/resources/locales/en/interim-applications-chd-daily-cause-list.json

```diff
@@ -0,0 +1,20 @@
+{
+    "title": "Interim Applications List (Chancery Division) Daily Cause List",
+    "heading": "Interim Applications List (Chancery Division) Daily Cause List",
+    "venueLine1": "Business and Property Courts Rolls Building",
+    "venueLine2": "7 Rolls Buildings, Fetter Ln, City of London",
+    "venueLine3": "London EC4A 1NL",
+    "openJusticeMessage1": "Parties should contact the clerk to the Interim Judge, ",
+    "openJusticeSeparator": ", ",
+    "openJusticeMessage2": " as early as possible.",
+    "openJusticeMessage3": "An application should not be listed before the Interim Applications Judge unless the overall time required to deal with the application is 2 hours or less. The 2 hour maximum includes the judge’s pre-reading time, the hearing of the application, delivery of judgment and time for dealing with costs. If the judge considers that the estimate will exceed the 2 hour limit it may be stood out of the interim applications list.",
+    "tableHeaders": [
+        "Judge",
+        "Time",
+        "Venue",
+        "Type",
+        "Case Number",
+        "Case Name",
```

Comment:
Can we make the table header casing for second word lower case, i.e. Case number and Case name to be consistent with other non-strategic list types